tests/test_reddit.py

```diff
@@ -0,0 +1,47 @@
+"""Unit tests for twitter.py.
```

Comment:
reddit

Reply:
ah! I was so close on the doc strings this time.